src/SignalR/server/Core/src/DefaultHubLifetimeManager.cs

```diff
@@ -80,12 +80,12 @@ public override Task RemoveFromGroupAsync(string connectionId, string groupName,
         }
 
         /// <inheritdoc />
-        public override Task SendAllAsync(string methodName, object?[]? args, CancellationToken cancellationToken = default)
+        public override Task SendAllAsync(string methodName, object?[] args, CancellationToken cancellationToken = default)
```

Comment:
Is this a change from .NET 5? There is an aspnetcore annoucement issue for nullability changes.

Reply:
This is an API change, however passing null would have thrown